specification/ml/_types/TrainedModel.ts

```diff
@@ -430,4 +481,6 @@ export class TrainedModelPrefixStrings {
    * String prepended to input at search
    */
   search?: string
+
+  none?: string
```

Comment:
Where is that coming from? Not seeing it in https://github.com/elastic/elasticsearch/blob/a827970d0bfefb6313964de5e059d24b6a1d7b12/x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/inference/TrainedModelPrefixStrings.java#L56-L66

Reply:
ah you're right I mixed up the class and the enum